### DISCUSSION TITLE
Media Player

### DISCUSSION THREAD

**Discussion by quenthal** [Blackymas/NSPanel_HA_Blueprint#1080] [Ideas]
Some customizable buttons for choosing playlists, radiochannels or similar would be cool.

There could be, let's say 3-9 buttons with either icons, numbers or customizable strings available for user with customizable media_content_Ids as values.. I think those could fit there quite nicely.

There might be some nice arrangement also for example if timeline and volume bar were replaced with buttons.

**Comment by edwardtfn** [comment]
Moved to #1082
